### PR TITLE
Added repo guidelines.

### LINKED ISSUE #3: Add contributing, license, and code of conduct.
All Builders Club projects need a Code of Conduct and Contributing guidelines.

Can probably "borrow" these from number-one?

### PR DESCRIPTION
**Pull request by exegeteio**
Fixes #3.

# Additions

Code of conduct.
Contributing.
License.
Bug report issue template.
Feature request issue template.
Bug fix pull request template.
Enhancement pull request template.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Community Guidelines
```
Comment (cmjchrisjones) [inline CONTRIBUTING.md:1]: Hey @exegeteio - this appears to be the same content as the Code Of Conduct file ;)

Comment (exegeteio) [reply CONTRIBUTING.md]: I knew it was too easy.  I copied over top of what I wrote on accident.  Should be updated.  Good catch.  👍

**Review by cmjchrisjones — APPROVED** [review]
Looks alright to me 👍

**Review by MichaelJolley — APPROVED** [review]

**Comment by MichaelJolley** [comment]
@all-contributors please add @exegeteio for documentation

**Pull request merged**